test/utils/sesar_api_test.py

```diff
@@ -0,0 +1,87 @@
+
+import inspect
+from pytest import raises
+
+from sample_uploader.utils.sesar_api import (
+    _get_igsn_endpoint,
+    retrieve_sample,
+)
+
+
+def start_test():
+    testname = inspect.stack()[1][3]
+    print('\n*** starting test: ' + testname + ' **')
+
+
+def fail_retrieve_sample(igsn, error_msg, error_type, contains=True):
```

Comment:
Nit: I always try to keep assertions to within the top level test functions themselves (I.e. this function could return a Boolean and the assertion could be made on the result), I'm not sure what standard practice is at KBase.